build/android/app/src/main/java/com/facebook/igl/sample/SampleView.java

```diff
@@ -162,13 +162,14 @@ public ConfigChooser(SampleLib.BackendVersion version) {
     public EGLConfig chooseConfig(EGL10 egl, EGLDisplay display) {
       final int EGL_OPENGL_ES2_BIT = 4;
 
-      // Set ourselves a strict configuration: RGBA8888, 16-bit depth buffer, no stencil.
+      // Set ourselves a strict configuration: RGBA8888, 16-bit depth buffer, 8-bit stencil buffer.
```

Comment:
Please add a parameter to control this. By default, the stencil buffer should be disabled.